src/main/java/harmonised/pmmo/config/codecs/AttributeModifier.java

```diff
@@ -0,0 +1,19 @@
+package harmonised.pmmo.config.codecs;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+public record AttributeModifier(
```

Comment:
Rename to `MobModifier` so as not to clash with the vanilla `AttributeModifer` class